yt_dlp/extractor/ard.py

```diff
@@ -640,6 +658,12 @@ def _real_extract(self, url):
             'description': description,
             'timestamp': unified_timestamp(player_page.get('broadcastedOn')),
             'series': try_get(player_page, lambda x: x['show']['title']),
+            'thumbnail': try_get(player_page, lambda x: x['image']['src'].replace('{width}', '1280')),
         })
         info.update(self._ARD_extract_episode_info(info['title']))
+        if not info.get('thumbnail'):
+            webpage = self._download_webpage(url, display_id, fatal=False) or ''
+            info['thumbnail'] = (
+                self._og_search_thumbnail(webpage, default=None)
+                or self._html_search_meta('thumbnailUrl', webpage, default=None))
```

Comment:
None of the test cases need/cover this block. Do we still want to keep it as an additional fallback ?

Reply:
Shouldn't the new thumbnail code go here?

In this code the newly discovered `.image.src` always overrides the `.mediaCollection._previewImage` (from `_parse_media_info()`), even if the the latter is valid and the former isn't.

Reply:
At least in the test cases it’s never overridden

Reply:
Either `.mediaCollection._previewImage` should be used as the primary or fallback value, or, if it can be determined that it's always blank (or never a valid URL), the corresponding query term can be removed from the GQL request.

Reply:
You are right, I found a video where _previewImage is set:

https://www.ardmediathek.de/video/tatort/love-is-pain-oder-team-dortmund-ermittelt/das-erste

![image](https://user-images.githubusercontent.com/36509607/233866820-1652c978-b4ba-43d3-86dd-0a11f36690c9.png)

Reply:
Added that as the primary value now, so we never break the working thumbnails